src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -256,63 +176,35 @@ class areaLinksTransCapaNumberOfTSretriever : public areaNumberOfTSretriever
         {
             const auto& link = *(it->second);
             to_return.push_back(link.directCapacities.timeSeries.width);
-            to_return.push_back(link.indirectCapacities.timeSeries.width);
         }
         return to_return;
     }
-    uint getGeneratedTimeSeriesNumber() override
-    {
-        return 1;
-    }
 };
 
 class IntraModalConsistencyChecker
 {
 public:
     IntraModalConsistencyChecker(const TimeSeriesType ts,
-                                 const array<bool, timeSeriesCount>& isTSintramodal,
-                                 const array<bool, timeSeriesCount>& isTSgenerated,
                                  areaNumberOfTSretriever* tsCounter,
                                  Study& study) :
-     tsCounter_(tsCounter), study_(study), nbTimeseries_(0)
+        tsCounter_(tsCounter), study_(study)
     {
-        int indexTS = ts_to_tsIndex.at(ts);
-        isTSintramodal_ = isTSintramodal[indexTS];
-        isTSgenerated_ = isTSgenerated[indexTS];
         tsTitle_ = ts_to_tsTitle.at(ts);
     }
     ~IntraModalConsistencyChecker() = default;
-    bool check();
+    bool checkTSconsistency();
     uint getTimeSeriesNumber()
     {
         return nbTimeseries_;
     }
 
 private:
-    bool checkTSconsistency();
-
-private:
-    bool isTSintramodal_;
-    bool isTSgenerated_;
     areaNumberOfTSretriever* tsCounter_;
     Study& study_;
-    uint nbTimeseries_;
+    uint nbTimeseries_ = 0;
     string tsTitle_;
 };
 
-bool IntraModalConsistencyChecker::check()
-{
-    if (isTSintramodal_ && not isTSgenerated_)
```

Comment:
**Explanations** : we don't have to know if TS are generated or not anymore, that why this function is removed